client2/src/game/gui/AccountPanel.js

```diff
@@ -0,0 +1,92 @@
+import PanelTemplate from "./PanelTemplate";
```

Comment:
This has already been converted to a component, file not needed.

Reply:
Removed at https://github.com/Arcanorum/dungeonz/pull/99/commits/559e46b0dab5d1a050e33f2b0faa246d76354a20